docs/dev/release.md

```diff
@@ -1,4 +1,70 @@
- What to do before every release?
+# Build Windows Installation:
```

Comment:
I don't think this file `release.md` is needed.

Reply:
Fixed